collector/host_inventory.py

```diff
@@ -57,7 +54,7 @@ def _retrieve_hosts(headers: dict) -> dict:
     return dict(results=results, total=total)
 
 
-def worker(_: str, source_id: str, dest: str, b64_identity: str) -> None:
+def worker(_: str, source_id: str, dest: str, misc: dict) -> None:
```

Comment:
Nit pick: `misc` sounds too generic for an important parameter that handles all account/identity related things

Reply:
any good suggestions?

Reply:
`acct_info` ?

Reply:
that'd do as well.

Using `misc` because I anticipate we may need to add more parameters going forward.  But let me honor your current concern instead of my future worries 😄